cypress/integration/createTrackedEntity.cy.js

```diff
@@ -75,37 +76,82 @@ const setUpTable = () => {
 
     selectProgramForTE(program.programName)
 
-    cy.getBySel('dimension-item-eventDate').should('not.exist')
-
+    // Check the correct time dimensions are displayed, and with program/stage specific name
     cy.getBySel(`dimension-item-${program.id}.enrollmentDate`).contains(
         program[DIMENSION_ID_ENROLLMENT_DATE]
     )
-
-    cy.getBySel('dimension-item-scheduledDate').should('not.exist')
-
     cy.getBySel(`dimension-item-${program.id}.incidentDate`).contains(
         program[DIMENSION_ID_INCIDENT_DATE]
     )
-
     cy.getBySel('dimension-item-lastUpdated').contains(
         program[DIMENSION_ID_LAST_UPDATED]
     )
+    cy.getBySel('dimension-item-eventDate').should('not.exist')
+    cy.getBySel('dimension-item-scheduledDate').should('not.exist')
 
+    // Add a program data dimension
     clickAddRemoveProgramDataDimension(programDataDimensionName)
 
-    // selectFixedPeriod({ label: periodLabel, period: TEST_FIX_PE_DEC_LAST_YEAR }) // FIXME: Time dimensions aren't implemented yet
+    const januaryThisYear = {
+        type: 'Monthly',
+        year: `${getCurrentYearStr()}`,
+        name: `January ${getCurrentYearStr()}`,
+    }
+
+    // Add a time dimension
+    selectFixedPeriod({
+        label: periodLabel,
+        period: januaryThisYear,
+    })
+
+    // Check the chip and tooltip content
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseover')
+    cy.getBySel('layout-chip-tooltip-content').contains(
+        `January ${getCurrentYearStr()}`
+    )
+    cy.getBySelLike('layout-chip').contains(periodLabel).trigger('mouseout')
+    cy.getBySel('layout-chip-tooltip-content').should('not.exist')
+
+    // Add another time dimension
+    // Check that the already selected period is not available to select
```

Comment:
This should ideally checked after `Update` has been clicked and/or after the AO has been saved. That's when the metadata gets manipulated.